packages/contracts/src/dollar/BondingShareV2.sol

```diff
@@ -32,26 +33,17 @@ contract BondingShareV2 is ERC1155, ERC1155Burnable, ERC1155Pausable {
 
     // ----------- Modifiers -----------
     modifier onlyMinter() {
-        require(
-            manager.hasRole(manager.UBQ_MINTER_ROLE(), msg.sender),
-            "Governance token: not minter"
-        );
+        require(manager.hasRole(manager.UBQ_MINTER_ROLE(), msg.sender), "Governance token: not minter");
         _;
     }
 
     modifier onlyBurner() {
-        require(
-            manager.hasRole(manager.UBQ_BURNER_ROLE(), msg.sender),
-            "Governance token: not burner"
-        );
+        require(manager.hasRole(manager.UBQ_BURNER_ROLE(), msg.sender), "Governance token: not burner");
         _;
     }
 
     modifier onlyPauser() {
-        require(
-            manager.hasRole(manager.PAUSER_ROLE(), msg.sender),
-            "Governance token: not pauser"
-        );
+        require(manager.hasRole(manager.PAUSER_ROLE(), msg.sender), "Governance token: not pauser");
```

Comment:
Looks like the formatter settings were changed. Perhaps you can restore the old settings and format again?

I'm assuming that the current formatter is not actually loading the settings as I noticed that [.prettierrc still has the `printWidth: 80`](https://github.com/ubiquity/ubiquity-dollar/blob/d72efd688b202bae8fca2b05dd715832aac51056/.prettierrc#L13) Solidity formatting setting, yet it looks like its wider now.

Reply:
Right now, we're using default `fmt` setting natively supported by `forge`. I am not 100% sure if `forge fmt` supports customized prettier setting. but will check it soon and keep u updated.

Reply:
i've found how to setup prettier rule in forge docs (https://book.getfoundry.sh/reference/config/formatter?highlight=fmt#formatter)

updated in https://github.com/ubiquity/ubiquity-dollar/pull/261/commits/b0fdaeb4806258ee7f79e3c202e0e8001e23111a